### PR TITLE
[BPF] fix rendering ipv6 iptables rules

### PR DESCRIPTION
**Pull request by tomastigera**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Review by fasaxc — APPROVED** [review]

File: felix/fv/bpf_test.go [hunk 1]
```diff
@@ -462,6 +462,7 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 					felix.Exec("conntrack", "-L")
 					felix.Exec("calico-bpf", "policy", "dump", "cali8d1e69e5f89", "all", "--asm")
 					if testOpts.ipv6 {
+						felix.Exec("conntrack", "-L", "-f", "ipv6")
```
Comment (aaaaaaaalex) [inline felix/fv/bpf_test.go:465]: Could this generate log spam?

**Review by aaaaaaaalex — APPROVED** [review]
Just one question above

File: felix/fv/bpf_test.go (hunk 1 above)
Comment (tomastigera) [reply felix/fv/bpf_test.go:465]: that is only when  test crashes to dump diags

**Comment by tomastigera** [comment]
unrelated flake

**Pull request merged**